src/components/ShowMore/index.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Text, View} from 'react-native';
+import _ from 'underscore';
+import Button from '@components/Button';
+import * as Expensicons from '@components/Icon/Expensicons';
+import useLocalize from '@hooks/useLocalize';
+import * as NumberFormatUtils from '@libs/NumberFormatUtils';
+import styles from '@styles/styles';
+import themeColors from '@styles/themes/default';
+
+const propTypes = {
+    /** A number of current showed items */
+    currentCount: PropTypes.number,
+
+    /** A number of total items */
+    totalCount: PropTypes.number,
+
+    /** A handler fires when button has been pressed */
+    onPress: PropTypes.func.isRequired,
+};
+
+const defaultProps = {
+    currentCount: undefined,
+    totalCount: undefined,
+};
+
+function ShowMore({currentCount, totalCount, onPress}) {
+    const {translate, preferredLocale} = useLocalize();
+
+    const shouldShowCounter = _.isNumber(currentCount) && _.isNumber(totalCount);
+
+    return (
+        <View style={styles.alignItemsCenter}>
+            {shouldShowCounter && (
+                <Text style={[styles.mb2, styles.textLabelSupporting]}>
+                    {`${translate('common.showing')} `}
+                    <Text style={styles.textStrong}>{currentCount}</Text>
+                    {` ${translate('common.of')} `}
+                    <Text style={styles.textStrong}>{NumberFormatUtils.format(preferredLocale, totalCount)}</Text>
+                </Text>
+            )}
+
+            <View style={[styles.w100, styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter]}>
+                <View style={[styles.shortTermsHorizontalRule, styles.flex1, styles.mr0]} />
+
+                <Button
+                    style={styles.mh0}
+                    small
+                    shouldShowRightIcon
+                    iconFill={themeColors.icon}
+                    iconRight={Expensicons.DownArrow}
+                    text={translate('common.showMore')}
+                    accessibilityLabel={translate('common.showMore')}
+                    onPress={onPress}
+                />
+
```

Comment:
NAB: are all these white spaces necessary here?

Reply:
I will remove them. Thank you!

Reply:
Done - e2d780d.